sunpy/coordinates/tests/test_transformations.py

```diff
@@ -96,7 +96,7 @@ def test_hcrs_hgs():
     earth_hgs = earth_hcrs.transform_to(HeliographicStonyhurst)
 
     # The HGS longitude of the Earth should be zero within numerical error
-    assert quantity_allclose(earth_hgs.lon % (360*u.deg), 0*u.deg, atol=1e-12*u.deg)
+    assert quantity_allclose(earth_hgs.lon % (360*u.deg), 0*u.deg, atol=1*u.deg)
```

Comment:
I think that just adding 180 to it will make this whole thing a lot more reliable.
```suggestion
    # Do the comparison against 180 to avoid Longitude wrapping issues.
    assert quantity_allclose(earth_hgs.lon + 180*u.deg, 180*u.deg, atol=1e12*u.deg)
```